.github/workflows/load-test.yml

```
@@ -0,0 +1,108 @@
+name: Load Test
+
+on:
+  schedule:
+    - cron: "0 0 * * 1" # Run every Monday at 00:00
+  workflow_dispatch:
+    inputs:
+      test_id:
+        type: string
+        description: "Identifier label for Datadog metrics"
+        default: "billing-relay-load-test"
+      k6_test_path:
+        type: string
+        description: "Path to load test files"
+        default: "test/load/*.js"
+      k6_flags:
+        type: string
+        description: "Additional k6 flags"
+      billing_relay_env_url:
+        type: string
+        description: "URL of the billing relay environment"
+        default: "https://billingrelay.qa.bitwarden.pw"
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  get-secrets:
+    name: Get secrets
+    runs-on: ubuntu-24.04
+    permissions:
+      id-token: write
+    outputs:
+      dd_api_key: ${{ steps.get-kv-secrets.outputs.dd-api-key }}
+      billing_paypal_webhook_key: ${{ steps.get-kv-secrets.outputs.billing-paypal-webhook-key }}
+    steps:
+      - name: Azure Login
```

```suggestion
      - name: Log in to Azure
```
I always suggest that these be action-focused.

---

What do you mean by "action-focused"? The title right now is essentially the action name, `azure-login` -> `Azure Login`.

---

Verbs basically -- completing an action.

---

Ah ok… action != GitHub action in this case 🫠